sdk/test-utils/testing-recorder-new/karma.conf.js

```diff
@@ -1,7 +1,10 @@
 // https://github.com/karma-runner/karma-chrome-launcher
+const { relativeRecordingsPathForBrowser } = require("@azure-tools/test-recorder-new");
 process.env.CHROME_BIN = require("puppeteer").executablePath();
 require("dotenv").config({ path: "./.env" });
 
+process.env.RECORDINGS_RELATIVE_PATH = relativeRecordingsPathForBrowser();
```

Comment:
This is needed to let proxy-tool know where to generate the recordings.

Reply:
Mocha supports `this.file` property in node and the same is undefined in the browser.

Reply:
So the old recorder doesn't need the relative path when saving recordings?

Reply:
It is required for node, but not for the browsers because we use karma plugin in the old recorder to generate the recordings at the same location as the base path(project entry point).

Reply:
fair